src/api/comp_metric.yaml

```diff
@@ -12,11 +12,11 @@ arguments:
     direction: input
     required: true
   - name: "--input_integrated_left"
-    __merge__: file_integrated.yaml
+    __merge__: file_integrated_left.yaml
```

Comment:
Would it be possible to find clearer names for these? It's not obvious to me what "left" and "right" and it would be great to be as clear as possible for anyone who might contribute in the future.

Reply:
The data is essentially divided into two halves. We can call it `left` and `right`, `fold1` and `fold2`, `left_half` and `right_half`, ...